src/main/kotlin/uk/co/craigbass/pratura/boundary/AddShippingAddress.kt

```diff
@@ -0,0 +1,17 @@
+package uk.co.craigbass.pratura.boundary
+
+import com.madetech.clean.usecase.SynchronousUseCase
+import uk.co.craigbass.pratura.boundary.AddShippingAddress.Request
+
+interface AddShippingAddress : SynchronousUseCase<Request, Unit> {
+  data class Request(
+    val name: String,
+    val companyName: String?,
+    val addressLine1: String,
+    val addressLine2: String?,
```

Comment:
Yeah in theory though people would probably put in stuff like

Blue door  <-- specific delivery instructions
The Conifers <-- Fuzzy house name
123 Fake St <-- actual line 1

And then say with Stripe you'd accidentally end up chopping off the actual address line 1 and having card declines

I can see the appeal of more flexibility though so hmm